app/imdb_tools.py

```diff
@@ -231,11 +230,13 @@ def tmdb_get_shows(config_path, plex, data, is_list=False):
 
     return matched, missing
 
-
 def tvdb_get_shows(config_path, plex, data, is_list=False):
     config_tools.TraktClient(config_path)
-    if not isinstance(data, int):
-        raise ValueError("| Config Error: TVDb ID: {} is invalid it must be an integer".format(data))
+    try:
```

Comment:
What was the issue?

Reply:
just in case the number is longer then an int can be the regex is a more complete solution